### ISSUE TITLE
USDT back testing error

### ISSUE THREAD

**Issue by dtb11288** [gcarq/freqtrade#397]
* Python Version: 3.6
  * Branch: Master
  * Last Commit ID: [70f2aed0a73fb1583b0f869fb7b0432ef9c9f068](https://github.com/gcarq/freqtrade/commit/70f2aed0a73fb1583b0f869fb7b0432ef9c9f068)
 
## Problem
*Error when running testing with USDT*

### Steps to reproduce:

  1. config:   "stake_currency": "USDT",  "pair_whitelist": [ "USDT_BTC", "USDT_XRP" ],
  2. (.env) > bin/freqtrade backtesting --realistic-simulation                                                                                                     
  

### Relevant code exceptions or logs:

  ```
2018-01-17 23:26:24,720 - freqtrade.optimize.backtesting - INFO - Using config: config.json ...
2018-01-17 23:26:24,721 - freqtrade.misc - INFO - Validating configuration ...
2018-01-17 23:26:24,724 - freqtrade.optimize.backtesting - INFO - Using ticker_interval: 5 ...
2018-01-17 23:26:24,724 - freqtrade.optimize.backtesting - INFO - Using local backtesting data (using whitelist in given config) ...
2018-01-17 23:26:24,724 - freqtrade.optimize - INFO - Download the pair: "USDT_BTC", Interval: 5 min
2018-01-17 23:26:26,291 - freqtrade.optimize.backtesting - INFO - Using stake_currency: USDT ...
2018-01-17 23:26:26,291 - freqtrade.optimize.backtesting - INFO - Using stake_amount: 10 ...
2018-01-17 23:26:26,291 - freqtrade.optimize.backtesting - INFO - Using max_open_trades: 5 ...
Traceback (most recent call last):
  File "bin/freqtrade", line 4, in <module>
    main()
  File "/data/workspace/freqtrade/freqtrade/main.py", line 412, in main
    args.func(args)
  File "/data/workspace/freqtrade/freqtrade/optimize/backtesting.py", line 179, in start
    preprocessed = preprocess(data)
  File "/data/workspace/freqtrade/freqtrade/optimize/__init__.py", line 67, in preprocess
    for pair, pair_data in tickerdata.items()}
  File "/data/workspace/freqtrade/freqtrade/optimize/__init__.py", line 67, in <dictcomp>
    for pair, pair_data in tickerdata.items()}
  File "/data/workspace/freqtrade/freqtrade/analyze.py", line 34, in parse_ticker_dataframe
    .drop('BV', 1) \
  File "/data/workspace/freqtrade/.env/lib/python3.6/site-packages/pandas/core/generic.py", line 2530, in drop
    obj = obj._drop_axis(labels, axis, level=level, errors=errors)
  File "/data/workspace/freqtrade/.env/lib/python3.6/site-packages/pandas/core/generic.py", line 2562, in _drop_axis
    new_axis = axis.drop(labels, errors=errors)
  File "/data/workspace/freqtrade/.env/lib/python3.6/site-packages/pandas/core/indexes/base.py", line 3744, in drop
    labels[mask])
ValueError: labels ['BV'] not contained in axis
  ```

**Comment by kryofly** [comment]
I dont think the USD as a basecurrency is supported.
But I think it would be a good idea in this case, because it would connect btc to the very important fiat.

**Comment by dtb11288** [comment]
seem like it's ok to run live, only backtest goes down, I just want to get backtest result before going further.
trading with USDT might feel safer than BTC in my opinion.

**Comment by kryofly** [comment]
Ok, I was probably wrong, and USDT is supported as a base currency, but perhaps not as well tested.
I'll try backtest with that pair as soon as I can.

**Comment by vertti** [comment]
This is probably just a bug, where Bittrex might not return the BaseVolume (BV) for USDT like it does for BTC. We don't need the BV value for anything so we just need to cleanly skip that.

**Comment by glonlas** [comment]
Agree with @vertti.

I will promote my last PR #400. It will make easier to add or remove indicator that are not compatibles.

**Comment by glonlas** [comment]
This issue will be totally fixed in develop branch once #428 will be merged.
